CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Lotus changelog
 
 # UNRELEASED
+Add `EthGetBlockReceipts` RPC method to retrieve transaction receipts for a specified block. This method allows users to obtain the receipts of all transactions included in a given Ethereum block. ([filecoin-project/lotus#12478](https://github.com/filecoin-project/lotus/pull/12478))
```

Comment:
```suggestion
Add `EthGetBlockReceipts` RPC method to retrieve transaction receipts for a specified block. This method allows users to obtain Ethereum format receipts of all transactions included in a given tipset as specified by its Ethereum block equivalent. ([filecoin-project/lotus#12478](https://github.com/filecoin-project/lotus/pull/12478))
```

... something like that, just trying to avoid "Ethereum block" as if this is a separate concept, it'll give output even if there are no FEVM transactions.